src/Angor/Client/Pages/Create.razor

```diff
@@ -1199,6 +1201,44 @@
         }
     }
 
+    private async Task RecalculateFee(decimal customFeeRate)
+    {
+        if (!passwordComponent.HasPassword())
+        {
+            showCreateModal = false;
+            notificationComponent.ShowErrorMessage("Wallet password expired");
+            return;
+        }
+
+        try
+        {
+            var words = await passwordComponent.GetWalletAsync();
+
+            // Create a custom fee estimation with the provided rate
+            var customFeeEstimation = new FeeEstimation
+                {
+                    FeeRate = (int)customFeeRate,
+                    Confirmations = 1 // Custom fees typically assume fast confirmation
```

Comment:
we cannot assume this, what does this matter what number we put here? is it only presentation?